api/src/main/java/uk/gov/dhsc/htbhf/claimant/repository/ClaimLazyLoader.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.dhsc.htbhf.claimant.repository;
+
+import uk.gov.dhsc.htbhf.claimant.entity.Claim;
+
+import java.util.UUID;
+
+/**
+ * Clients should not refer to this interface directly - use ClaimRepository instead.
+ */
+public interface ClaimLazyLoader {
```

Comment:
Can you explain the reasoning behind needing this now? Just wanted some further background before approving...

Reply:
It will be needed for my subtask - by the upcoming `CreateNewPaymentCycleJob`, which will need to get a reference to the claim to pass to the `PaymentService`. 
It is admittedly coming in the wrong order as it's not needed by the next PR but the one after. My next PR will be the scheduler and an empty implementation of `CreateNewPaymentCycleJob`, followed by implementing the job - which will need this.

Reply:
Ok, that's fine thing, just didn't have any context in this PR.